ai_feedback/image_processing.py

```diff
@@ -114,25 +116,15 @@ def process_image(args, prompt: dict) -> tuple[str, str]:
                 "{context}", "```\n" + context + "\n```"
             )
         if prompt.get("include_image_size", False):
-            submission_image_paths = read_submission_images(OUTPUT_DIRECTORY, question)
-            submission_image_path = submission_image_paths[
-                0
-            ]  # Only consider one image per question
```

Comment:
The restriction that we only consider one image per question is important to document, both in the code here (as a comment) and in the README